addons/website/static/tests/tours/link_tools.js

```diff
@@ -37,9 +37,18 @@ wTourUtils.registerWebsitePreviewTour('link_tools', {
         content: "Click on the newly created link",
         trigger: 'iframe .s_text_image a[href="http://odoo.com"]:contains("odoo.com")',
     },
+    {
+        content: "Label value should contain odoo.com",
+        trigger: '#o_link_dialog_label_input',
+        run: () => {
+            if ($('#o_link_dialog_label_input').val() !== 'odoo.com') {
+                throw new Error('Label value should contain odoo.com');
```

Comment:
I'm confused. Why throw an error for this ? When I look at the previous value of the step, I expect the tour to simply fail if the input value is not correct.